server/cmwell-cons/src/main/scala/ctl.scala

```diff
@@ -349,8 +349,18 @@ abstract class Host(user: String,
                  host: String,
                  sudo: Boolean,
                  sudoer: Option[Credentials]) {
-    if (sudo)
-      command(s"""echo -e '$content' | sudo tee $path > /dev/null""", host, true, sudoer)
+    if (sudo) {
+      val tempFile = File.createTempFile("pre-", ".txt")
+      val fullPath = tempFile.getAbsolutePath
+      val fileName = tempFile.getName
+      val writer = new PrintWriter(tempFile)
+      writer.write(content)
+      writer.close()
+      _rsync(fullPath, "~", s"${sudoer.get.name}@$host", sudo = false)
```

Comment:
Change to:
command(s"""echo $$'$content' > $path""", host, false)

Reply:
Tried it. It worked for the `60-cassandra.conf` but no for the `cmwell-runner`. Leaving it as it is now.